.gitignore

```diff
@@ -136,7 +136,7 @@ tabular/src/autogluon/tabular/version.py
 text/src/autogluon/text/version.py
 multimodal/src/autogluon/multimodal/version.py
 eda/src/autogluon/eda/version.py
-
+experiments/
```

Comment:
Is this folder newly introduced by this PR?

Reply:
oh I was running some experiments with the code and thus had this folder within the repo. Should I remove it entirely?

Reply:
If it is not part of repo and bother other users, we can remove it for now

Reply:
Sure!